dicetable.py

```diff
@@ -0,0 +1,327 @@
+
+
+class dicetable(object):
```

Comment:
if i change everything now (functions, class variables, function variables, etc...)according to those rules, it'll take maybe 1/2 hour.  if i wait till later, it'll be a true pain.  any reason not to change it all?